magefile.go

```diff
@@ -342,13 +352,18 @@ func patchPackageOperatorManager(scheme *k8sruntime.Scheme, obj *unstructured.Un
 		panic(fmt.Errorf("converting to Deployment: %w", err))
 	}
 
-	var packageOperatorManagerImage string
+	var (
+		packageOperatorManagerImage string
+		remotePhasePackageImage     string
+	)
 	if len(os.Getenv("USE_DIGESTS")) > 0 {
 		// To use digests the image needs to be pushed to a registry first.
-		mg.Deps(mg.F(Build.PushImage, "package-operator-manager"))
+		mg.Deps(mg.F(Build.PushImage, "package-operator-manager", remotePhasePackageName))
```

Comment:
I think `PushImage` only takes one argument. Maybe this is what you mean `mg.Deps(mg.F(Build.PushImage, "package-operator-manager"), mg.F(Build.PushImage, pkoPackageName))`?

Reply:
Right